internal/iam/command_rbac_role_binding_list.go

```diff
@@ -59,57 +59,57 @@ func (c *roleBindingCommand) newListCommand() *cobra.Command {
 				Code: "confluent iam rbac role-binding list --principal User:u-123456 --inclusive",
 			},
 			examples.Example{
-				Text: "List the role bindings for the current user at the environment scope and its nested scopes:",
+				Text: "List the role bindings for the current user with the environment scope and subscopes:",
 				Code: "confluent iam rbac role-binding list --current-user --environment env-123456 --inclusive",
 			},
 		)
 	} else {
 		cmd.Example = examples.BuildExampleString(
 			examples.Example{
-				Text: "Only use the `--resource` flag when specifying a `--role` with no `--principal` specified. If specifying a `--principal`, then the `--resource` flag is ignored. To list role bindings for a specific role on an identified resource:",
-				Code: "confluent iam rbac role-binding list --kafka-cluster $KAFKA_CLUSTER_ID --role DeveloperRead --resource Topic",
+				Text: `Only use the "--resource" flag when specifying a "--role" with no "--principal" specified. If specifying a "--principal", then the "--resource" flag is ignored. To list role bindings for a specific role on an identified resource:`,
```

Comment:
Please revert the changes on this line. We use code highlighting when flags and command names are wrapped in backticks, just like Slack

Reply:
fixed

Reply:
It doesn't look like these changes were added

Reply:
fixed